docs/resources/connection_client.md

```diff
@@ -1,27 +1,29 @@
 ---
 page_title: "Resource: auth0_connection_client"
 description: |-
-  With this resource, you can manage enabled clients on a connection.
+  With this resource, you can enable a single client on a connection.
 ---
 
 # Resource: auth0_connection_client
 
-With this resource, you can manage enabled clients on a connection.
+With this resource, you can enable a single client on a connection.
+
+!> To prevent issues, avoid using this resource together with the `auth0_connection_clients` resource.
 
 ## Example Usage
 
 ```terraform
 resource "auth0_connection" "my_conn" {
   name     = "My-Auth0-Connection"
   strategy = "auth0"
-  # Avoid using the enabled_clients = [...],
```

Comment:
This is from the time we allowed managin the enabled_clients on the connection resource. Should not be here any more.